src/mkdocs_include_markdown_plugin/process.py

```diff
@@ -117,7 +117,17 @@
 MARKDOWN_HTML_IMAGE_REGEX = re.compile(
```

Comment:
`MARKDOWN_HTML_IMAGE_REGEX` and `MARKDOWN_HTML_SOURCE_REGEX` are practially the same. Can you merge them into one using somehing like `<(?:img)|(?:source)`?

Reply:
Agreed. Merged them.